### PR TITLE
New package: perl-Text-ASCIITable

### PR DESCRIPTION
**Pull request by chinarulezzz**

### CODE REVIEW

**Comment by chinarulezzz** [comment]
@maxice8 Thanks for the review.

Updated :-)

**Comment by maxice8** [comment]
@chinarulezzz i meant to put noarch= above where wrksrc= was not to move wrksrc= to below noarch=

**Comment by chinarulezzz** [comment]
Should I do anything?

File: srcpkgs/perl-Text-ASCIITable/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'perl-Text-ASCIITable'
+pkgname=perl-Text-ASCIITable
+version=0.22
+revision=1
+build_style=perl-module
```
Comment (newbluemoon) [inline srcpkgs/perl-Text-ASCIITable/template]: The build_style has to be `perl-ModuleBuild`. Then the cross builds won’t fail.

**Comment by newbluemoon** [comment]
> Should I do anything?

Move
```
noarch=yes
wrksrc=...
```
directly below `revision`.

File: srcpkgs/perl-Text-ASCIITable/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'perl-Text-ASCIITable'
+pkgname=perl-Text-ASCIITable
+version=0.22
+revision=2
```
Comment (newbluemoon) [inline srcpkgs/perl-Text-ASCIITable/template]: The revision of a <i>new</i> package should always be 1. ;)

File: srcpkgs/perl-Text-ASCIITable/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+noarch=yes
+wrksrc="${pkgname/perl-/}-${version}"
+build_style=perl-ModuleBuild
+hostmakedepends="perl perl-Module-Build"
+makedepends="perl"
+depends="${makedepends}"
```
Comment (newbluemoon) [inline srcpkgs/perl-Text-ASCIITable/template]: Might be a matter of taste, but if it only depends on perl I think using `${makedepends}` here looks a bit ugly. Just `depends="perl"` suffices.

**Comment by chinarulezzz** [comment]
Done. Sorry guys for so much disturb :)

**Comment by newbluemoon** [comment]
The cross failure seems to be Travis related. Worked fine locally. @chinarulezzz you could rebase and force push again to trigger a fresh Travis build.

**Comment by maxice8** [comment]
On Sat, Nov 17, 2018 at 08:42:03AM -0800, newbluemoon wrote:
> The cross failure seems to be Travis related. Worked fine locally. @chinarulezzz you could rebase and force push again to trigger a fresh Travis build.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4637#issuecomment-439630175

No need i can trigger a rebuild here.

**Comment by newbluemoon** [comment]
> No need i can trigger a rebuild here.

Cool. Didn’t know that was possible. :)

**Comment by newbluemoon** [comment]
Ahh, now I see what’s wrong: @chinarulezzz you removed

    makedepends="perl"

but it’s essential for cross builds to succeed!

**Comment by chinarulezzz** [comment]
updated

**Pull request merged**